hosts/hosts.go

```diff
@@ -420,6 +420,15 @@ func (h *Host) GetExtraArgs(service v3.BaseService) map[string]string {
 	}
 }
 
+func (h *Host) GetExtraArgsArray(service v3.BaseService) map[string][]string {
+	switch {
+	case h.OS() == "windows" && len(service.WindowsExtraArgsArray) > 0:
+		return service.WindowsExtraArgsArray
+	default:
+		return service.ExtraArgsArray
+	}
+}
```

Comment:
nit: switch is not necessary here.
```suggestion
	if h.OS() == "windows" && len(service.WindowsExtraArgsArray) > 0 {
		return service.WindowsExtraArgsArray
	}

	return service.ExtraArgsArray
}
```